libnavui-dropin/src/main/java/com/mapbox/navigation/dropin/map/MapViewBinder.kt

```diff
@@ -21,47 +18,57 @@ import com.mapbox.navigation.dropin.map.longpress.RoutePreviewLongPressMapCompon
 import com.mapbox.navigation.dropin.map.marker.MapMarkersComponent
 import com.mapbox.navigation.dropin.map.scalebar.ScalebarComponent
 import com.mapbox.navigation.dropin.navigationview.NavigationViewContext
-import com.mapbox.navigation.ui.app.internal.Store
 import com.mapbox.navigation.ui.app.internal.navigation.NavigationState
-import com.mapbox.navigation.ui.app.internal.routefetch.RoutePreviewAction
-import com.mapbox.navigation.ui.app.internal.routefetch.RoutePreviewState
-import com.mapbox.navigation.ui.app.internal.routefetch.RoutesAction
 import com.mapbox.navigation.ui.base.lifecycle.UIBinder
 import com.mapbox.navigation.ui.maps.internal.ui.LocationComponent
 import com.mapbox.navigation.ui.maps.internal.ui.LocationPuckComponent
 import com.mapbox.navigation.ui.maps.internal.ui.RouteArrowComponent
 import com.mapbox.navigation.ui.maps.internal.ui.RouteLineComponent
-import com.mapbox.navigation.ui.maps.internal.ui.RouteLineComponentContract
 import com.mapbox.navigation.ui.maps.route.arrow.model.RouteArrowOptions
 import com.mapbox.navigation.ui.maps.route.line.model.MapboxRouteLineOptions
-import kotlinx.coroutines.FlowPreview
-import kotlinx.coroutines.flow.Flow
-import kotlinx.coroutines.flow.combine
 
 @ExperimentalPreviewMapboxNavigationAPI
-@FlowPreview
-internal class MapBinder(
-    private val context: NavigationViewContext,
-    private val binding: MapboxNavigationViewLayoutBinding,
-    private val mapView: MapView
-) : UIBinder {
-
-    init {
-        mapView.compass.enabled = false
+abstract class MapViewBinder : UIBinder {
+
+    companion object {
+        /**
+         * Default Info Panel Binder.
+         */
+        fun defaultBinder(): MapViewBinder = MapboxMapViewBinder()
+    }
```

Comment:
Let's follow [recommended coding conventions](https://kotlinlang.org/docs/coding-conventions.html#class-layout) and update this class layout by moving this companion object to the bottom.